applications/rag/frontend/container/static/script.js

```diff
@@ -142,6 +142,42 @@ function toggleNlpFilterSection(nlpEnabled) {
     }
 }
 
+function updateNLPValue() {
+    const rangeInput = document.getElementById('nlp-range');
+    const valueDisplay = document.getElementById('nlp-value');
+
+    // Function to update the slider's display value and color
+    const updateSliderAppearance = (value) => {
+        // Update the display text
+        valueDisplay.textContent = value;
+
+        // Determine the color based on the value
+        let color;
+        if (value <= 25) {
+            color = '#EA4335';
+        } else if (value <= 50) {
+            color = '#FBBC05';
+        } else if (value <= 75) {
```

Comment:
can we have comments on what those colors are?

Reply:
Sure, will do in next PR